jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+
+For a complete list of all API v2 enpoints, please refer to the [API v2 Reference Guide](https://circleci.com/docs/api/v2/#circleci-api), which contains a detailed description of each individual endpoint, as well as information on required and optional parameters, HTTP status and error codes, and code samples you may use in your workflows.
+
+### New Endpoints
+
+The table below describes the new endpoints that have been added to the CircleCI API for this updated v2 version.
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`GET /workflow/:id ` | This endpoint enables users to return an individual Workflow based on the `id` parameter being passed in the request
+`GET /workflow/:id/jobs` | This endoint enables users to retrieve all Jobs associated with a specific workflow, based on its unique `id`.
+`GET /project/:project_slug`  | This endpoint enables users to retrieve a specific Project by its unique slug.
+`POST /project/:project_slug/pipeline` | This endpoint enables users to retrieve an individual project by its unique slug.
+`GET /pipeline/:id` | This endpoint enables users to retrieve an individual pipeline, based on the `id` passed in the request.
+`GET /pipeline/:id/config`  | This endpoint enables users to retrieve the configuration of a specific pipeline.
+`GET /project/:project_slug/pipelines/[:filter]`  | This endpoint enables users to retrieve the most recent set of pipelines for a Project.
+
+### Deprecated Endpoints
+{:.no_toc}
+
+For this updated API v2 release, several endpoints have been deprecated, which are listed in the table below.
+
+Endpoint       | Description
+-----------|-----------------------------------------------------
+`POST /project/:vcs-type/:username/:project`  | This endpoint allowed users to trigger a new build.
+`POST /project/:vcs-type/:username/:project/build` | This endpoint enables users to trigger a new build by project.
+`DELETE /project/:vcs-type/:username/:project/build-cache` | This endpoint enabled users to clear the project cache for a specific project.
+`GET /recent-builds` | This endpoint enabled users to retrieve an array of recent builds.
+
+## API v1.1 Support and EOL
+
+With the release of API v2, there will be two different versions of the CircleCi API that you can use. Although CircleCI recommends you use API v2, you may still use API v1.1 if you wish; however, please note that API v2 includes several powerful features (e.g. support for pipelines) that are unavailable in API v1.1.
+
+For a short period of time, both versions of the CircleCI API (v1.1 & v2) will be available. CircleCI expects to eventually End-Of-Life (EOL) API v1.1 and discontinue support for this API version in the near future. Guidance on when the CircleCI API v1.1 will be discontinued will be communicated at a future date.
```

Comment:
suggesting we use `For a period of time`

Reply:
fixed